src/Component/Filter/AttributeCombo/AttributeCombo.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
```

Comment:
I'm not sure if it makes a difference but i'd prefer `import React from 'react';`.

Reply:
Does not work for me. The module doesn't seem to have a standard export. And using ``import { React } from 'react';`` also does not work because there is no exported member ``React`` in the module. 
I also just discovered `` import * as React from 'react'; `` in all documentations / tutorials I read so far.